### DISCUSSION TITLE
Pages got missing on linksta.cc

### DISCUSSION THREAD

**Discussion by diogocsc** [JulianPrieber/littlelink-custom#630] [Issues]
Hi,
I had created two pages on linksta.cc with different emails, and they are now not found.
When I try to login with one of my google accounts, I get a 500 server error.

I was considering linksta.cc for another projects as well, though I don't know how much I can trust this service.
Is this just a trial service that erases everything after some time? If so, how much time?

**Comment by JulianPrieber** [comment]
Linksta.cc is a site that's here to stay. If you are having trouble logging into your account you can reach our support at info@linkstack.org.
